github/Team.py

```diff
@@ -263,6 +271,18 @@ def edit(
         )
         self._useAttributes(data)
 
+    def get_teams(self):
+        """
+        :calls: `GET /teams/:id/teams <https://developer.github.com/enterprise/2.17/v3/teams/#list-child-teams>`_
+        :rtype: :class:`github.PaginatedList.PaginatedList` of :class:`github.Team.Team`
+        """
+        return github.PaginatedList.PaginatedList(
```

Comment:
Doesn't this request still need a custom Accept header?

Reply:
Not anymore :-)
You can test against some org.
e.g.
```
curl -s -X GET https://api.github.com/orgs/testorggshefer/teams/aa/teams -H "Accept: application/json" -H "Content-Type: application/json" -u username:password
[
  {
    "name": "aaa",
    "id": 3593746,
    "node_id": "MDQ6VGVhbTM1OTM3NDY=",
    "slug": "aaa",
    "description": "aaa",
    "privacy": "closed",
    "url": "https://api.github.com/teams/3593746",
    "html_url": "https://github.com/orgs/testorggshefer/teams/aaa",
    "members_url": "https://api.github.com/teams/3593746/members{/member}",
    "repositories_url": "https://api.github.com/teams/3593746/repos",
    "permission": "pull",
    "parent": {
      "name": "aa",
      "id": 3593733,
      "node_id": "MDQ6VGVhbTM1OTM3MzM=",
      "slug": "aa",
      "description": "aa",
      "privacy": "closed",
      "url": "https://api.github.com/teams/3593733",
      "html_url": "https://github.com/orgs/testorggshefer/teams/aa",
      "members_url": "https://api.github.com/teams/3593733/members{/member}",
      "repositories_url": "https://api.github.com/teams/3593733/repos",
      "permission": "pull"
    }
  }
]

```